XcodeServerSDK/Server Entities/DeviceSpecification.swift

```diff
@@ -19,12 +19,14 @@ public class DevicePlatform : XcodeServerEntity {
         case iOS_Simulator = "com.apple.platform.iphonesimulator"
         case OSX = "com.apple.platform.macosx"
         case watchOS = "com.apple.platform.watchos"
-        case watchOS_Simulator = "com.apple.platform.watchsimulator"
+        case watchOS_Simulator = "com.apple.platform.watchsimulator" // Why is it here?
```

Comment:
Because simulators are different "platforms" - this is related to Xcode Server, which treats simulators and devices as different platforms. 

On that note, is there a simulator platform for tvOS as well? I would expect there will be...